package.json

```diff
@@ -79,7 +79,7 @@
     "gulp-util": "^3.0.0",
     "is-docker": "^2.2.1",
     "istanbul": "^0.4.5",
-    "karma": "^6.3.2",
+    "karma": "^6.4.3",
```

Comment:
Oh doh, I didn't see this and the package.lock change before approving. @bhasker-ddh could you remove both of these so there are no longer conflicts before merging?

Reply:
Can you check it now @ncolletti

Reply:
@bhasker-ddh The PR still has the `karma` version bump and changes to `package-lock`, could you try removing these again?